Item(by='bunkerbewohner', descendants=None, kids=[24675404, 24675286], score=None, time=1601764388, title=None, item_type='comment', url=None, parent=24674855, text='Has been down for me in Berlin, Germany, for an hour or so.<p>Trying to log in via web gives me this error:<p>Method: auth.sendCode\nResult: {&quot;_&quot;:&quot;rpc_error&quot;,&quot;error_code&quot;:420,&quot;error_message&quot;:&quot;FLOOD_WAIT_438&quot;}<p>Maybe a localized DDoS attack?')